### DISCUSSION TITLE
How much time does this program take for completion?

### DISCUSSION THREAD

**Discussion by throwaway12345678910111** [Discord-OTP-Forcer/Discord-OTP-Forcer#61] [Community Support] [answered]
Im using this to get into a lost account but I want to know what is the longest amount of time it could take to get into the ACC I also want to know is there any way of speeding up the attempts

**Comment by Prastal** [comment, marked as answer]
To have an approximately 95%  chance of getting the 1 in a million correct you would need 2,995,732 attempts which with an average of 11.5 seconds a request it would take 57.238 weeks, so around a year. To have a 50% chance of getting it correct it requires 693,147 attempts which would be 13.178 weeks in ideal conditions. And there probably isnt a way to speed it out since discord would rate limit you.

**Reply by realCraymotherlol** [reply]
Say isn’t it a possibility to put on a vpn to somewhat speed it up?

**Reply by Prastal** [reply]
It works with my own VPN running off of a VPS but it seems to be net limiting based on the token. I found that i had rate limits on every 2 attempts rather than 3.

**Reply by Krakenne** [reply]
Isn't it 10 in 1 000 000 every 30 sec or so? If it is then how come that in 13 weeks you have 50:50 to hit it or not when if it refresh every 30 sec you have still 10 in 1 000 000 if Im not ccorrect please explain it to me like Im 5.

**Comment by Johndoehackie123** [comment]
Took me a few minutes.

**Reply by XiLime** [reply]
im really curious
im on 1.5 hours but well aware it will take a LONG time.

**Reply by salemtheenby** [reply]
> im really curious im on 1.5 hours but well aware it will take a LONG time.

Xilime has it worked yet for you?

**Reply by Johndoehackie123** [reply]
> Fr? it worked for you?

I can't remember lol I don't even remember posting this. It most likely worked on a dummy account or I just posted it may work in a certain amount of time.

**Reply by XiLime** [reply]
@salemtheenby not yet but its still cooking

**Comment by salemtheenby** [comment]
Wait Im confused some people are saying over a year some people say a few minutes mines been running for about 1 hour already on my laptop  so it works it just seems to take a while

**Reply by salemtheenby** [reply]
So how long approximately does it actually take like between
